journal/modernized/08-chapter-5.adoc

```diff
@@ -23,8 +23,8 @@ and as his father approached him,
 his eye rested on Petersburg with such an intimation
 that there his Divine Master would send him,
 that, said he, in relating the circumstance many years afterwards to an intimate friend,
-'I never afterwards doubted f he however kept the matter entirely to himself,
-believing that when the right time should come, way would be made for him.
+"I never afterwards doubted if he however kept the matter entirely to himself,
```

Comment:
This is an example of where I think it would be helpful for me to send you a PDF scan of the original book. The text in the PDF actually says: ```'I never afterwards doubted;' he however kept...'```  So the semicolon and the single quote got turned into an f when we were converting the PDF to Word.  The word "if" is not in the text at all. :)